### DISCUSSION TITLE
Auto-closing quotes

### DISCUSSION THREAD

**Discussion by R0315** [dotnet/interactive#2766] [Q&A] [answered]
Hello!

I've noticed that in my .NET Interactive notebook when I type a quote it doesn't auto-close the quote.  Oddly, when I first open the notebook this functionality works.  However, once the .NET Interactive kernel starts it stops working.  Does anyone know how I can get this working?  I've checked:

Settings for this feature are "Always"
Updated the kernel

**Comment by jonsequitur** [comment, marked as answer]
This is a known issue: #2631.

**Reply by R0315** [reply]
Oh thank you! I did search for quote issues but didn't find anything.  I've never used github before though, I probably just didn't search very well.
